components/org.wso2.carbon.identity.sso.saml.ui/src/main/java/org/wso2/carbon/identity/sso/saml/ui/SAMLSSOUIConstants.java

```diff
@@ -51,6 +51,8 @@
 
     public static final String SAML_SSO_SIGNING_ALGORITHM = "signingAlgorithm";
     public static final String SAML_SSO_DIGEST_ALGORITHM = "digestAlgorithm";
```

Comment:
Isn't it good to update Licence Header to the 2017

Reply:
Since this is an existing class I think we don't need to update it.